<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_DadosCliente</class>
 <widget class="QWidget" name="frm_DadosCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #eaf2f8;
    border-radius: 8px;
}</string>
  </property>
  <widget class="QLabel" name="lbl_nome">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nome">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_celular">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Celular: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cidade">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Cidade:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffebee; 
    border: 2px solid #ffcdd2;
    border-radius: 10px;
    color: #b71c1c; 
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image: url(:/newPrefix/Img/cancelar.png);
    background-repeat: no-repeat;
    background-position: center;
    padding-left: 40px;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #ffcdd2;
    border-color: #e57373;
    color: #d32f2f; 
}

QPushButton:pressed {
    background-color: #e57373;
    border-color: #b71c1c;
    padding-left: 44px;
    padding-top: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cadastrar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 2px solid #d1c4b2;
    border-radius: 12px;
    color: #5a5a5a;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image:url(:/newPrefix/Img/cadastrar.png); 
    background-repeat: no-repeat;
    background-position: center;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #f2ebe0;
    border-color: #c4b49e;
}

QPushButton:pressed {
    background-color: #e8dfcf; 
    border-color: #b39b8d; 
    padding-left: 12px; 
    padding-top: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_celular">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="inputMask">
    <string>(00) 0 0000-0000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_cidade">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="kk.qrc"/>
 </resources>
 <connections/>
</ui>
